src/quo2/components/buttons/button/type_values.cljs

```diff
@@ -0,0 +1,186 @@
+(ns quo2.components.buttons.button.type-values
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo2.theme]))
+
+(defn custom-color-type
+  [customization-color]
+  {:icon-color           colors/white
+   :icon-secondary-color colors/white-opa-70
+   :label-color          colors/white
+   :background-color     (colors/custom-color
+                          customization-color
+                          50)})
+
+(defn get-value-from-type
+  [{:keys [customization-color theme type background pressed?]}]
+  (cond
+    (= type :primary) (custom-color-type customization-color)
+    (= type :positive) (custom-color-type :success)
+    (= type :danger) (custom-color-type :danger)
+    ;; TODO Remove type blurred - https://github.com/status-im/status-mobile/issues/16535
```

Comment:
Maybe we can delete all these TODO comments since we have issues already?

Reply:
sure 👌